web/src/desktop_integration.js

```diff
@@ -62,4 +64,29 @@ if (window.electron_bridge !== undefined) {
             error,
         });
     });
+
+    $(() => {
+        $(document).on("click", "#open-self-hosted-billing", (event) => {
```

Comment:
This is the key piece, replaces the on click behavior for that "Plan management" `<a href>` to actually make an API request in the background. Kind of hacky, not sure if there's a cleaner way to do it?

Reply:
This seems basically fine, though we should do `desktop_integration.initialize()` as the way to setup the click handler, like we do for basically every other module. 

@andersk thoughts?

Reply:
Ah, so adding that in `initialize_everything` instead of doing the `$(() => {` thing directly?

Reply:
Yep!

Reply:
Okay, pushed with that approach.